sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

```diff
@@ -544,6 +545,45 @@ public void DuplicateKeysInLogRecordAttributesAndLogScope2()
             Assert.Equal(duplicateScopeValue2, actualScopeValue);
         }
 
+        [Fact]
+        public void ValidateCustomEvent()
+        {
+            // Arrange.
+            var logRecords = new List<LogRecord>(1);
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
```

Comment:
not really necessary, but could you add a test with a `LogRecordProcessor` and see that it can inspect this, just like any other normal ILogger logs.. (it should be, but i am just making sure everything is covered!)

Reply:
its for creating the batch later.